src/main/webapp/index.html

```diff
@@ -1,13 +1,33 @@
 <!DOCTYPE html>
-  <html>
+<html>
 
-    <head>
-	    <meta charset="UTF-8">
-	    <title></title>
-    </head>
+<head>
+	<meta charset="UTF-8">
+	<title>Givr</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="index_script.js"></script>
+</head>
 
-  <body>
+<body>
+	<div id="help-near-me-content">
+		<div id="site-title">Givr</div>
+		<div id="nav-bar">
+			<a href="index.html">Help Near Me</a>
+			<a href="">Login</a>
+		</div>
+		<div id="zipcode-form">
```

Comment:
Consider the following, it is up to you whether you want to have the page as is now as a placeholder and do what I describe below in a later PR

Do you want to make this zipcode-form and filter-button-area a combined JS class?

ie, html:
<div id="search-organizations"></div>

in js:
class SearchOrganizations {
  constructor(elem: HTMLElement) {
     this.mainElement = elem;
     // create the zipcode input, the filter input
     // create the organization list div
     this.requestAndDisplayOrganizations();
  } 

  requestAndDisplayOrganizations() {
    const data = await requestOrgsFromCurrentInput();
    displayData(data);
  }
}

Reply:
Then you can re-use this "widget" in other pages.

Reply:
doing this in my second pull request